libraries/exoplayer_hls/src/main/java/androidx/media3/exoplayer/hls/playlist/HlsPlaylistParser.java

```diff
@@ -374,7 +411,39 @@ private static HlsMultivariantPlaylist parseMultivariantPlaylist(
         int roleFlags = isIFrameOnlyVariant ? C.ROLE_FLAG_TRICK_PLAY : 0;
         int peakBitrate = parseIntAttr(line, REGEX_BANDWIDTH);
         int averageBitrate = parseOptionalIntAttr(line, REGEX_AVERAGE_BANDWIDTH, -1);
+        String videoRange = parseOptionalStringAttr(line, REGEX_VIDEO_RANGE, variableDefinitions);
         String codecs = parseOptionalStringAttr(line, REGEX_CODECS, variableDefinitions);
+        String supplementalCodecsStrings =
+            parseOptionalStringAttr(line, REGEX_SUPPLEMENTAL_CODECS, variableDefinitions);
+        String supplementalCodecs = null;
+        String supplementalProfiles = null;  // i.e. Compatibility brand
+        if (supplementalCodecsStrings != null) {
+          String[] supplementalCodecsString = Util.split(supplementalCodecsStrings, ",");
+          if (!supplementalCodecsString[0].isEmpty()) {
```

Comment:
Just want to confirm which two lines you refer to? Is it Line 421 and Line 422? so this code will be changed to
`        if (supplementalCodecsStrings != null) {`
`          String[] supplementalCodecsString = Util.splitAtFirst(supplementalCodecsStrings, ",");`
`          // TODO: Support more than one element`
`          String[] codecsAndProfiles = Util.split(supplementalCodecsString[0], "/");`
`          supplementalCodecs = codecsAndProfiles[0];`
`          if (codecsAndProfiles.length > 1) {`
`            supplementalProfiles = codecsAndProfiles[1];`
`          }`
`        }`

Reply:
I was thinking about this line in addition to the one you already proposed:
```
String[] codecsAndProfiles = Util.split(supplementalCodecsString[0], "/");
```
But maybe that doesn't work because we need to split at more than one `/` ?

Reply:
Yes, for current implementation, using splitAtFirst is OK. According to the definition in [IETF](https://datatracker.ietf.org/doc/html/draft-pantos-hls-rfc8216bis-15), the supplementalCodecsString could be very complex. It is up to you to decide how to implement this part of code.

Reply:
Thanks for the confirmation!